src/containers/StructurePage/resourceComponents/ResourceItemLink.tsx

```diff
@@ -9,24 +9,23 @@
 import { Link } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
 import styled from '@emotion/styled';
-import { colors, mq, breakpoints } from '@ndla/core';
+import { colors, fonts, mq, breakpoints } from '@ndla/core';
 import { toEditArticle, toLearningpathFull } from '../../../util/routeHelpers';
 
-const StyledH1 = styled.h1<{ isVisible?: boolean }>`
+type FontSizeType = 'small' | 'medium';
+
+const StyledH1 = styled.h1<{ isVisible?: boolean; size?: FontSizeType }>`
   font-style: ${props => !props.isVisible && 'italic'};
   color: ${props => (!props.isVisible ? colors.brand.grey : colors.brand.primary)};
   text-transform: none;
   letter-spacing: 0;
   margin: 0;
   display: inline;
-  font-size: 18px;
-  font-size: 1rem;
-  line-height: 1.44444;
-  font-weight: 500;
+  ${props => fonts.sizes(props.size === 'small' ? 16 : 18, 1.44444)};
+  font-weight: ${props => (props.size === 'small' ? fonts.weight.normal : 500)};
```

Comment:
Hvordan ser `fonts.semibold` ut her? Skal vi innføre en ny font-weight? Isåfall er det sikkert fint å legge den inn i `fonts ` i frontend-packages.